specification/maps/data-plane/Creator/readme.md

```diff
@@ -46,7 +46,6 @@ input-file:
   - preview/2.0/dataset.json
   - preview/2.0/dwgconversion.json
   - preview/2.0/featurestate.json
-  - preview/1.0/spatial.json
```

Comment:
Why is this being removed?

Reply:
hi TJ! We have a set of services that fall under Azure Maps, and some that fall under Azure Maps Creator. Trying to fix the nav in the docs so that it presents itself appropriately.

https://docs.microsoft.com/en-us/rest/api/maps/

Azure Maps should have
- Data
- Evelation
- Geolocation
- Render
- Route
- Search
- Spatial
- Timezone
- Traffic
- Weather

Azure Maps Creator should have only V2 (v1 was preview and we deprecated that version in Sept 2021):
- Alias
- Conversion
- Dataset
- Feature State
- Tileset
- WFS

-